source/examples/xgboost-randomforest-gpu-hpo-dask/notebook.ipynb

```diff
@@ -0,0 +1,926 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "workflows/hpo",
+     "library/dask-ml",
+     "library/cuml",
+     "cloud/aws/ec2",
```

Comment:
I think we could include tags for all the other single node pages too given that this isn't specifically about EC2
```suggestion
     "cloud/aws/ec2",
     "cloud/azure/azure-vm",
     "cloud/gcp/compute-engine",
     "cloud/ibm/virtual-server",
```